cclib/parser/turbomoleparser.py

```diff
@@ -176,7 +176,9 @@ def extract(self, inputfile, line):
         #   1   c           x   0.00000  0.00001  0.00000 -0.01968 -0.04257  0.00001
         #                   y  -0.08246 -0.08792  0.02675 -0.00010  0.00000  0.17930
         #                   z   0.00001  0.00003  0.00004 -0.10350  0.11992 -0.00003
-        if 'NORMAL MODES and VIBRATIONAL FREQUENCIES (cm**(-1))' in line:
+        # raman.out can have
+        # if 'NORMAL MODES and VIBRATIONAL FREQUENCIES (cm**(-1))' in line:
+        if 'a o f o r c e' in line:  

```

Comment:
Can this be changed to `if 'NORMAL MODES and VIBRATIONAL FREQUENCIES (cm**(-1))' in line or 'a o f o r c e' in line:`? The tests are failing because the previous output files had the former

Reply:
Oh, sorry. This was not supposed to get into the pull request.